.appveyor.yml

```diff
@@ -1,41 +1,43 @@
 init:
- # -ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
- 
+# -ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+
+version: 1.0.0.{build}
+
 configuration:
   - Release
   - Debug
 
 image:
   - Visual Studio 2015
-  - Visual Studio 2013
 
 platform:
   - x64
+  - x86
 
 build_script:
-  # build pthread
-  - git clone https://github.com/GerHobbelt/pthread-win32.git C:/pthread-src
-  - ps: "if ( $Env:APPVEYOR_BUILD_WORKER_IMAGE -eq 'Visual Studio 2013' ) { $VS_VERSION='2013' } else { $VS_VERSION='2015' }"
-  - ps: "msbuild C:/pthread-src/pthread.${VS_VERSION}.sln /p:Configuration=$Env:CONFIGURATION /p:Platform=x64"
-  - ps: mkdir C:/pthread-win32
-  - ps: mkdir C:/pthread-win32/include
-  - ps: mkdir C:/pthread-win32/lib
-  - ps: cp C:/pthread-src/*.h C:/pthread-win32/include
-  - ps: cp C:/pthread-src/bin/x64_MSVC${VS_VERSION}.${Env:CONFIGURATION}/pthread_lib.lib C:/pthread-win32/lib
+  # prep shared variables
+  - ps: if ( $Env:PLATFORM -eq 'x86' ) { $DEVENV_PLATFORM = 'Win32' } else { $DEVENV_PLATFORM = 'x64' }
+  - ps: if ( $Env:PLATFORM -eq 'x86' ) { $FOLDER_PLATFORM = 'win32' } else { $FOLDER_PLATFORM = 'win64' }
+  - ps: if ( $Env:APPVEYOR_BUILD_WORKER_IMAGE -eq 'Visual Studio 2013' ) { $VS_VERSION='2013' } else { $VS_VERSION='2015' }
+  - ps: $env:VS_VERSION = $VS_VERSION
+  # get pthreads
+  - ps: nuget install cinegy.pthreads-$FOLDER_PLATFORM -version 2.9.1.17
   # build SRT
-  - ps: if ( $VS_VERSION -eq "2013" ) { $CMAKE_GENERATOR = "Visual Studio 12 2013 Win64" } else { $CMAKE_GENERATOR = "Visual Studio 14 2015 Win64" }
-  - ps: if ( $VS_VERSION -eq "2013" ) { $ENABLE_UNITTESTS = "OFF" } else { $ENABLE_UNITTESTS = "ON" }
+  - ps: if ( $VS_VERSION -eq '2015' -and $Env:PLATFORM -eq 'x86' ) { $CMAKE_GENERATOR = 'Visual Studio 14 2015' }
+  - ps: if ( $VS_VERSION -eq '2015' -and $Env:PLATFORM -eq 'x64' ) { $CMAKE_GENERATOR = 'Visual Studio 14 2015 Win64' }
+  - ps: if ( $VS_VERSION -eq '2015' ) { $ENABLE_UNITTESTS = 'ON' } else { $ENABLE_UNITTESTS = 'OFF' }
```

Comment:
Without VS 2013 this can be simplified.

Reply:
Yes it can - i left it more complex because I have no idea if we'd need to support multiple VS versions in the future (e.g. if we added VS2019).

Reply:
We have a history in git. The simpler, the better.

Reply:
OK - I'll happily do it; by that logic shall we accept the premise of VS2013 disappearing and if we need it back we can just revise from GIT? It's really just a small part of the PR; the addition of 32-bit and using pthreads as a binary is the more meaningful aspect.

Reply:
This is no longer topical now VS2013 is back, so will resolve.